### ISSUE TITLE
Bug with IndicatorTypes.EMAIL

### ISSUE THREAD

**Issue by Te-k** [AlienVault-OTX/OTX-Python-SDK#41]
If I do `res = otx.get_indicator_details_full(IndicatorTypes.EMAIL, args.EMAIL)` I get the following error : 
```
File "/home/user/env/analysis3/lib/python3.5/site-packages/OTXv2.py", line 490, in get_indicator_details_full
    for section in indicator_type.sections:
TypeError: 'NoneType' object is not iterable
```

Which makes sense because IndicatorTypes.EMAIL is the only one not having sections at all in `IndicatorTypes.py` while `get_indicator_details_full` use this to request information.

I have this bug with python 3.5.2 and the pip package 1.2 (but I am pretty sure this bug is the master code)

**Comment by rustybrooks** [comment]
You're right.  I've fixed this in master and it'll go into the next patch release.
